core/embed/prodtest/main.c

```diff
@@ -101,6 +124,66 @@ static void vcp_printf(const char *fmt, ...) {
   vcp_puts("\r\n", 2);
 }
 
+static void vcp_write_as_hex(uint8_t *data, uint16_t len) {
```

Comment:
`vcp_println_hex` then?

Reply:
e14114794714d5f8982a2c765fbe99d8d72c159e and e14114794714d5f8982a2c765fbe99d8d72c159e